docs/team/seantehds.md

```diff
@@ -8,40 +8,41 @@ title: Teh De Sheng Sean's Project Portfolio Page
+* **New Feature:** Unmark Command: [#61](https://github.com/AY2324S1-CS2103T-T17-1/tp/pull/61)
     * What it does: Marks a task within the task list as incomplete.
     * Justifications: Allows the user to mark tasks as incomplete to allow the task manager to keep track of
       tasks and whether they are finished or not.
     * Highlights: The task becomes opaque when it is marked as incomplete.
     * Credits: Sebastian for implementing the Status class and field necessary for tracking whether the
       task has been completed or not.
-* New Component: Deadline: [#78](https://github.com/AY2324S1-CS2103T-T17-1/tp/pull/78)
+* **New Component:** Deadline: [#78](https://github.com/AY2324S1-CS2103T-T17-1/tp/pull/78)
     * What it does: A class that encapsulates the specified deadline of a given task within TaskWise.
     * Justifications:
         * User: Allows users to track the deadlines of each task, which would be displayed along with the task within TaskWise.
         * Developers: By creating a Deadline class, the functionality and expected behaviour of the deadlines within each task would be easier to maintain as well as extend by future developers.
     * Credits: George for assisting with the creation of default deadlines for newly created tasks
 
-* **Enhancement to existing features**:
-    * Updated the Add Command to allow for variable arguments from the user: [#127](https://github.com/AY2324S1-CS2103T-T17-1/tp/pull/127)
-        * Before 1.3, the Add Command was only able to take in a description input from the user in order to create a task within TaskWise. This would result in a task being created with all other fields, like deadlines and priorities being initialised to their default values.
-        * As a group, we believed that the user experience would be enhanced if users were given an option to create a task with certain specified fields already initialised to values already given by the user's input.
-        * Therefore, I updated the Add Command from 1.2 to be able to take in arguments, and create a task with these fields already initialised to these given inputs.
+### Enhancement To Existing Features
+* Updated the Add Command to allow for variable arguments from the user: [#127](https://github.com/AY2324S1-CS2103T-T17-1/tp/pull/127)
+  * Before 1.3, the Add Command was only able to take in a description input from the user in order to create a task within TaskWise. This would result in a task being created with all other fields, like deadlines and priorities being initialised to their default values.
+  * As a group, we believed that the user experience would be enhanced if users were given an option to create a task with certain specified fields already initialised to values already given by the user's input.
+  * Therefore, I updated the Add Command from 1.2 to be able to take in arguments, and create a task with these fields already initialised to these given inputs.
```

Comment:
Hi May, could you change this line to be this instead? 

Therefore, I updated the Add Command from 1.2 to be able to take in additional arguments that were optional, and create a task with these fields already initialised to these given inputs.